features/z-index.yml

```
@@ -0,0 +1,11 @@
+name: z-index
+description: The `z-index` CSS property allows you to order overlapping elements, with higher values in front of or on top of lower values.
```

Nice description. One suggestion for brevity (lots of phrases like "allows you to…", "is used to…", "enables…" can be reduced to whatever verb immediately follows) and one for clarity:

```suggestion
description: The `z-index` CSS property orders overlapping elements, with higher values appearing in front of or on top of lower values.
```